client-app/src/pages/catalog/App.vue

```diff
@@ -1,12 +1,28 @@
 <template>
-  <h1>Hello world!</h1>
+  <div>
+    <h1>{{ searchCriteria }}</h1>
+    <h2>{{ searchQuery }}</h2>
+  </div>
 </template>
 <script lang="ts">
 import Vue from "vue";
 import Component from "vue-class-component";
+import { ProductSearchCriteria } from "@common/models/search/product-search-criteria.ts";
+import { ProductSearchQuery } from "@common/models/search/product-search-query.ts";
+import { QueryBuilder } from "@common/services/query-builder.service.ts";
 
 @Component({
   name: "Catalog"
 })
-export default class App extends Vue {}
+export default class App extends Vue {
+  get searchCriteria() {
+    const t1 = new URL(location.href).searchParams;
```

Comment:
what does t1 mean? I think the name "searchParams" will be a bit better for this :)

Reply:
This is temporary testing code, it shouldn't exist at all. Same for few comments below